taf/src/main/java/com/epam/tat/test/Constants.java

```diff
@@ -0,0 +1,10 @@
+package com.epam.tat.test;
+
+public class Constants {
```

Comment:
I would also make the constructor of this class private to forbid creating its instances.